.travis.yml

```diff
@@ -41,6 +41,11 @@ env:
 python:
   - "2.7"
   - "3.5"
+  - "pypy"
+
+matrix:
+  allow_failures:
```

Comment:
Merged.  BTW shouldn't we add also: 

- `Programming Language :: Python :: Implementation :: CPython` 

in addition to:

-  `Programming Language :: Python :: Implementation :: PyPy`

?